client/src/components/tests/TagField-test.js

```diff
@@ -1,88 +1,64 @@
 /* eslint-disable import/no-extraneous-dependencies */
-/* global jest, describe, beforeEach, it, expect, setTimeout, document */
-
-jest.mock('isomorphic-fetch');
+/* global jest, test, describe, beforeEach, it, expect, setTimeout, document */
 
 import React from 'react';
-import Enzyme, { shallow } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
 import { Component as TagField } from '../TagField';
-import Select from 'react-select';
-import AsyncSelect from 'react-select/async';
-import AsyncCreatableSelect from 'react-select/async-creatable';
-import CreatableSelect from 'react-select/creatable';
-import fetch from 'isomorphic-fetch';
-
-Enzyme.configure({ adapter: new Adapter() });
-
-describe('TagField', () => {
-  let props;
-
-  beforeEach(() => {
-    props = {
-      name: 'Test',
-      labelKey: 'label',
-      valueKey: 'value',
-      lazyLoad: false,
-      creatable: false,
-      multi: true,
-    };
-  });
-
-  describe('should render a Select component with type', () => {
-    it('Select', () => {
-      const wrapper = shallow(
-        <TagField {...props} />
-      );
-      expect(wrapper.find(Select).length).toBe(1);
-    });
-    it('Select.Creatable with creatable option', () => {
-      props.creatable = true;
-      const wrapper = shallow(
-        <TagField {...props} />
-      );
-      expect(wrapper.find(CreatableSelect).length).toBe(1);
-    });
-    it('Select.Async with lazyLoad option', () => {
-      props.lazyLoad = true;
-      const wrapper = shallow(
-        <TagField {...props} />
-      );
-      expect(wrapper.find(AsyncSelect).length).toBe(1);
-    });
-    it('Select.AsyncCreatable with both creatable and lazyLoad options', () => {
-      props.creatable = true;
-      props.lazyLoad = true;
-      const wrapper = shallow(
-        <TagField {...props} />
-      );
-      expect(wrapper.find(AsyncCreatableSelect).length).toBe(1);
-    });
-  });
+import { render, screen } from '@testing-library/react';
 
-  describe('with lazyLoad on and given a URL', () => {
```

Comment:
It no longer makes sense to test a the fetch request here using a mock as the AsyncSelectComponent we're passing in doesn't make a fetch